twill-core/src/main/java/org/apache/twill/internal/TwillRuntimeSpecification.java

```diff
@@ -72,7 +76,7 @@ public String getZkConnectStr() {
     return zkConnectStr;
   }
 
-  public RunId getTwillRunId() {
+  public RunId getTwillAppRunId() {
```

Comment:
There is only one `RunId`, why not just call if `runId` and `getRunId` for the field name and method name respectively?